experiments/tf_trainer/common/tfrecord_simple.py

```diff
@@ -48,12 +48,14 @@ def _input_fn_from_file(self, filepath: str) -> types.FeatureAndLabelTensors:
 
     parsed_dataset = dataset.map(
         self._read_tf_example, num_parallel_calls=multiprocessing.cpu_count())
+    return parsed_dataset
 
-    padded_shapes = ({
-        self._text_feature: [],
-    }, {label: [] for label in self._labels})
-    batched_dataset = parsed_dataset.prefetch(self._num_prefetch)
-    return batched_dataset
+    #padded_shapes = ({ self._text_feature: [] },
```

Comment:
What's with the commented code here? Remove?

Reply:
You need to keep some level of batching done, as mentioned in the email.
Bucket_by_sequence_length is not required.